packages/react-dev-overlay/src/internal/container/Errors.tsx

```diff
@@ -53,6 +53,14 @@ function getErrorSignature(ev: SupportedErrorEvent): string {
   return ''
 }
 
+function makeClickable(text: string): string {
+  // Regex Checks for http:// or https:// links
+  const link_regex = /(https?:\/\/)([^\s/$.?#].[^\s]*)/gi
```

Comment:
The capture groups are not needed to match:

```suggestion
  const link_regex = /https?:\/\/[^\s/$.?#].[^\s]*/gi
```

Reply:
Done! 8f57bbe